website/database.py

```diff
@@ -773,14 +778,30 @@ def get_quiz_stats(self, ids, start=None, end=None):
         data = [QUIZ_STATS.get_many({"id": i, "week": dynamo.Between(start_week, end_week)}) for i in ids]
         return functools.reduce(operator.iconcat, data, [])
 
-    def add_program_stats(self, id, level, number_of_lines, exception):
+    def add_program_stats(self, id, level, number_of_lines, exception, error_message=None):
         key = {"id#level": f"{id}#{level}", "week": self.to_year_week(date.today())}
-
         add_attributes = {"id": id, "level": level, "number_of_lines": number_of_lines}
+        program_stats = PROGRAM_STATS.get_many({"id": id, "week": self.to_year_week(date.today())})
```

Comment:
A little bit of a comment that this code is used in the new dashboard would be nice